py3status/modules/backlight.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+"""
+Display the current screen backlight level.
+
+Configuration parameters:
+    cache_timeout: How often we refresh this module in seconds (default: 10s)
```

Comment:
we don't need the `s` or the `:` in the default so `(default 10)`

`(default None)` and `(default '☼: {level}%')` for device and format would be good too